### PR TITLE
New module: EMBOSS_SEQRET

### LINKED ISSUE #1886: new module: EMBOSS/SEQRET
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by MillironX**
## PR checklist

Closes #1886

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/emboss/seqret/main.nf [hunk 1]
```diff
@@ -0,0 +1,37 @@
+process EMBOSS_SEQRET {
+    tag "$meta.id"
+    label 'process_low'
```
Comment (SPPearce) [inline modules/emboss/seqret/main.nf:3]: Can we use the new process_single tag here?

Comment (Emiller88) [inline modules/emboss/seqret/main.nf:3]:
```suggestion
    label 'process_single'
```

**Review by Emiller88 — COMMENTED** [review]
Agree with everything @SPPearce said. Awesome work on the testing!

File: modules/emboss/seqret/meta.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+name: "emboss_seqret"
+description: Reads in one or more sequences and writes them out again
```
[the remark this answers is not in the document]
Comment (MillironX) [reply modules/emboss/seqret/meta.yml]: Edited to say

> Reads in one or more sequences, converts, filters, or transforms them and writes them out again

How is that?

**Comment by SPPearce** [comment]
I'm happy with the code, but don't know why one of the tests is now failing on all 3 platforms.

**Comment by MillironX** [comment]
:facepalm: GFF includes a timestamp for time of conversion. Removed the md5 test for that.

**Review by SPPearce — APPROVED** [review]

**Pull request merged**